api/src/opentrons/server/endpoints/calibration/check.py

```diff
@@ -29,11 +29,31 @@ def _format_status(
     status = CalibrationSessionStatus(
         instruments=instruments,
         currentStep=current,
-        nextSteps=links,
-        sessionToken=session.token)
+        nextSteps=links)
     return status
 
 
+async def get_session(request):
+    """
+    GET /calibration/check/session
+
+    If a session exists, this endpoint will return the current status.
+
+    Otherwise, this endpoint will return a 404 with links to the post request.
+    """
+    session_type = request.match_info['type']
+    session_storage = request.app['com.opentrons.session_manager']
+    current_session = session_storage.sessions.get(session_type)
+    if current_session:
+        response = _format_status(current_session, request.app.router)
+        return web.json_response(text=response.json(), status=200)
+    else:
+        response = {
+            "message": "No check session exists. Please create one.",
+            "links": {"POST": "/calibration/check/session"}}
```

Comment:
```suggestion
            "links": {"createSession": f"/calibration/{session_type}/session"}}
```

I'm not sure how we should structure the required verb for a link here but I think we should keep the key as a short name, and if we want to add the verb it should be in the value somewhere